index.html

```diff
@@ -4,5 +4,6 @@
     <title> Welcome to XYZ</title>
   </head>
   <body>
+      <h1>Welcome to our shop!</h1>
```

Comment:
This is good but could you please add a message, "we are ready to serve you" and make the color blue